static/js/table.js

```diff
@@ -176,6 +176,11 @@ class Table {
     if (type !== 'start') return
     this.resultNotified = false
     document.getElementById('dealer-chip').style.display = 'block'
+    document.getElementById('player-chip').style.display = 'block'
+    let pname = getCookie('name')
```

Comment:
Mēs beigās paliksim pie cookies? Man liekas, ka ir ok, kaut ko iemācījāmies. Neiespringsim dikti uz to, ceru, ka mēs šo spēli vēl paturpināsim.

Bet par mainīgo, es laikam labāk šādi:
```js
const playerName = getCookie('name')
```